radix-engine/src/kernel/interpreters/scrypto_interpreter.rs

```diff
@@ -217,6 +217,10 @@ impl ExecutableInvocation for MethodInvocation {
             executor,
         ))
     }
+
+    fn payload_size(&self) -> usize {
+        self.args.len()
```

Comment:
Can we limit everything in the invocation?

Reply:
Added method name length to method invocation and function name, blueprint name, package address lengths to function invocation.